### ISSUE TITLE
NetworkOnMainThreadException when using socket appender with async appender

### ISSUE THREAD

**Issue by gauravdott** [tony19/logback-android#91]
Here is configuration in assets/logback.xml

```
<?xml version="1.0" encoding="utf-8"?>
<configuration debug="true">

    <appender name="SOCKET" class="ch.qos.logback.classic.net.SocketAppender">
        <!-- lazy initialization: don't open socket until 1st write -->
        <lazy>true</lazy>

        <remoteHost>###My host name here</remoteHost>
        <port>80</port>
        <reconnectionDelay>10000</reconnectionDelay>
        <includeCallerData>true</includeCallerData>

        <encoder>
            <pattern>%d{ISO8601} %p %t %c{0}.%M - %m%</pattern>
        </encoder>

    </appender>

    <appender name="ASYNC" class="ch.qos.logback.classic.AsyncAppender">
        <appender-ref ref="SOCKET" />
    </appender>

    <root level="info">
        <appender-ref ref="ASYNC" />
    </root>
    <root level="debug">
        <appender-ref ref="ASYNC" />
    </root>
    <root level="error">
        <appender-ref ref="ASYNC" />
    </root>
    <root level="warn">
        <appender-ref ref="ASYNC" />
    </root>
</configuration>
```

Also I am closing the logger context in onDestroy

```
@Override
    protected void onDestroy() {
        super.onDestroy();
        LoggerContext loggerContext = (LoggerContext) LoggerFactory.getILoggerFactory();
        loggerContext.stop();
    }
```
Can anybody suggest a solution?

**Comment by tony19** [comment]
This is a duplicate of #90.

Unrelated problem: your configuration contains too many `<root>` entries. You only need one. If you want your root logger to capture all logs, simply set the level to `debug`, which would include all log levels.
